t/sqlite.t

```diff
@@ -930,7 +931,7 @@ subtest 'Nested data structures' => sub {
 subtest 'Perform job in a running event loop' => sub {
   plan skip_all => 'Minion workers do not support fork emulation' if HAS_PSEUDOFORK;
   my $id = $minion->enqueue(add => [8, 9]);
-  Mojo::IOLoop->delay(sub { $minion->perform_jobs })->wait;
+  Mojo::Promise->new(sub { shift->($minion->perform_jobs) })->wait;
```

Comment:
This Mojo::Promise constructor requires Mojolicious 8.15. Instead of bumping the prereq it could just be done the same way as in this change in Mojo::Pg https://github.com/mojolicious/minion/commit/8abc075f42d52e4480c91d2745d151a3f8cfc62d

Reply:
Thanks for catching that!